File: replace.py
import os
import re

files_to_update = [
    'src/app/login/page.tsx',
    'src/app/settings/page.tsx',
    'src/app/layout.tsx',
    'src/components/layout/header.tsx',
    'src/components/layout/sidebar.tsx',
    'src/context/role-context.tsx',
    'src/lib/mock-data.ts',
    'package.json'
]

for file_path in files_to_update:
    if not os.path.exists(file_path):
        continue
    with open(file_path, 'r', encoding='utf-8') as f:
        content = f.read()

    # Replacements
    content = content.replace('teamos.ai', 'uef.edu.vn')
    content = content.replace('TeamOS AI', 'UEF Marcom')
    content = content.replace('TeamOS Workspace', 'UEF Marcom Workspace')
    content = content.replace('TeamOS', 'UEF Marcom')
    content = content.replace('teamos-role', 'uef-marcom-role')
    content = content.replace('teamos-ai', 'uef-marcom')

    with open(file_path, 'w', encoding='utf-8') as f:
        f.write(content)

print("Done replacing text.")

File: replace.js
const fs = require('fs');
const path = require('path');

const filesToUpdate = [
    'src/app/login/page.tsx',
    'src/app/settings/page.tsx',
    'src/app/layout.tsx',
    'src/components/layout/header.tsx',
    'src/components/layout/sidebar.tsx',
    'src/context/role-context.tsx',
    'src/lib/mock-data.ts',
    'package.json'
];

for (const filePath of filesToUpdate) {
    if (!fs.existsSync(filePath)) continue;
    let content = fs.readFileSync(filePath, 'utf-8');

    content = content.replace(/teamos\.ai/g, 'uef.edu.vn');
    content = content.replace(/TeamOS AI/g, 'UEF Marcom');
    content = content.replace(/TeamOS Workspace/g, 'UEF Marcom Workspace');
    content = content.replace(/TeamOS/g, 'UEF Marcom');
    content = content.replace(/teamos-role/g, 'uef-marcom-role');
    content = content.replace(/teamos-ai/g, 'uef-marcom');

    fs.writeFileSync(filePath, content, 'utf-8');
}

console.log("Done replacing text.");
